app/views/home/_event.html.erb

```diff
@@ -0,0 +1,10 @@
+<div>
```

Comment:
These are just some partials that I've set up for the new homepage. Please disregard them.